damien/models/evaluation.py

```diff
@@ -481,17 +481,11 @@ def merge_transient(
         default_form=None,
         default_evaluation_types=None,
     ):
-        transient_evaluation = cls(
+        transient_evaluation = saved_evaluation or cls(
```

Comment:
@lyttam What was your thought in equating the transient evaluation to the saved evaluation here? I'm not sure what all the effects would be but they might be unpredictable; most code in the evaluation model is expecting that transient evaluation will be constructed on the fly, optionally drawing from a saved object but not being identical to it.

Reply:
@pauline2k Initially I thought it was needed for the solution, but I just tried reverting those lines and the bug stayed fixed. I'll amend.